### PR TITLE
Small fixes in sessions 2.1 and 2.2

### PR DESCRIPTION
**Pull request by gootyfer**

### CODE REVIEW

**Comment by nasivuela** [comment]
LGTM!
I just replaced 'Virgen by 'Diosa'.

File: sprint_2/2_2_intro_a_la_programacion_2.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+[codepen-string-interpolation]: https://codepen.io/adalab/pen/wpPZvN?editors=0010
```
Comment (nasivuela) [inline sprint_2/2_2_intro_a_la_programacion_2.md]: @gootyfer This looks odd on the first line xD ¿Maybe a mistake?

Comment (gootyfer) [reply sprint_2/2_2_intro_a_la_programacion_2.md]: It's the declaration of a reference (link) used below in a example. I think it's OK

Comment (gootyfer) [reply sprint_2/2_2_intro_a_la_programacion_2.md]: @nasivuela you can embed the link to the codepen is you think is more consistent with the rest of the session.

**Pull request merged**